script.py

```diff
@@ -0,0 +1,59 @@
+import os
```

Comment:
Instead, add unit tests in `tests/keras/preprocessing/sequence_test.py`. Unit tests help demonstrate use cases.

Reply:
Added tests now